<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadedTimers</class>
 <widget class="QWidget" name="ThreadedTimers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThreadedTimers</string>
  </property>
  <widget class="QPushButton" name="buttonSet">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>163</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>83</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Timer 0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Timer 1</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>122</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000000000; </string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>9</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelT0">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>14</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Timer 0:	0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelT1">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>44</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Timer 1:	0</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSet</sender>
   <signal>clicked()</signal>
   <receiver>ThreadedTimers</receiver>
   <slot>set()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonStop</sender>
   <signal>clicked()</signal>
   <receiver>ThreadedTimers</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>stopTimer(Timer*)</signal>
  <slot>stop()</slot>
  <slot>set()</slot>
  <slot>tock()</slot>
 </slots>
</ui>
